packages/dataverse/docs/GET_STARTED.md

```diff
@@ -0,0 +1,512 @@
+returns the product of their values. In the spreadsheet analogy it would be like
+having two cells with two functions and third cell that contains a function that
+calculates the product of the previous two cells. Whenever the first two cells
+recalculate their value, the third cell will also do the same.
+
+Here's how we would solve this problem in `dataverse`:
+
+```typescript
+import {Box, prism} from '@theatre/dataverse'
+
+const widthB = new Box(1)
+const heightB = new Box(2)
+const padding = 5
+
+const widthWithPaddingD = widthB.derivation.map((w) => w + padding)
+const heightWidthPaddingD = heightB.derivation.map((h) => h + padding)
+
+const areaD = prism(() => {
+  return widthWithPaddingD.getValue() * heightWidthPaddingD.getValue()
+})
+
+console.log('area: ', areaD.getValue())
+widthB.set(10)
+console.log('new area: ', areaD.getValue())
+```
+
+### `usePrism()`
+
+You can also use derivations inside of React components with the `usePrism()`
+hook from the `@theatre/react` package, which accepts a dependency array for the
+second argument. All the values that are tracked should be included in the
+dependency array.
+
+> Note that all the changes of the derivations' values inside the callback
+> function (first argument of `usePrism()`) will be tracked, since `usePrism()`
+> uses the `prism()` function under the hood. However, if the derivations are
+> not provided in the dependency array of `usePrism()`, then it will not know if
+> the derivations themselves have changed. Don't worry if it sounds confusing,
+> we'll cover this behavior later.
+
+#### A simple example
+
+Here's a simple example: we have a Box that contains the width and height of a
+div (let's call it `panel`). Imagine that we want to have a button that changes
+the width of the `panel` to a random number when clicked.
+
+```typescriptreact
+import {Box} from '@theatre/dataverse'
+import {usePrism} from '@theatre/react'
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+// Set the original width and height
+const panelB = new Box({
+  dims: {width: 200, height: 100},
+})
+
+function changePanelWidth() {
+  const oldValue = panelB.get()
+  // Change `width` to a random number between 0 and 200
+  panelB.set({dims: {...oldValue.dims, width: Math.round(Math.random() * 200)}})
+}
+
+const Comp = () => {
+  const render = usePrism(() => {
+    const {dims} = panelB.derivation.getValue()
+    return (
+      <>
+        <button
+          style={{
+            display: 'block',
+            height: 25,
+            width: 200,
+          }}
+          onClick={() => changePanelWidth()}
+        >
+          Change the width
+        </button>
+        <div
+          style={{
+            display: 'block'
+            width: dims.width,
+            height: dims.height,
+            backgroundColor: '#bd6888',
+          }}
+        ></div>
+      </>
+    )
+  }, [panelB]) // Note that `panelB` is in the dependency array
```

Comment:
Yepp, but isn't it good practice to include it in the dependency array in case anyways? It is [mentioned later that `panelB` can be omitted](https://github.com/theatre-js/theatre/blob/20a26cbe050da60a506875482c9ba3fa632e1062/packages/dataverse/docs/GET_STARTED.md#the-dependency-array) from the dependency array.